packages/cli/src/sso/saml/routes/saml.controller.ee.ts

```diff
@@ -27,8 +27,10 @@ import { getSamlConnectionTestFailedView } from '../views/samlConnectionTestFail
 import { InternalHooks } from '@/InternalHooks';
 import url from 'url';
 import querystring from 'querystring';
+import { RequireGlobalScope } from '@/decorators/Scopes';
 
 @Service()
+@Authorized()
```

Comment:
just FYI: `Authorized` was meant to be a "temporary" solution until we have all endpoints migrated to a decorated controller class. 
IMO, We should make all controllers authorized by default (allowing only instance owner), with an option to pass in a `authScopes` option in `RestController`.

Reply:
@ivov What happened is I removed the `@Authorized` owner checks from some routes. Which caused them to fall into `authFreeRoutes` list. Which meant that the user object didn't exist on the request since it didn't go through the passport middleware. So I was getting 401s from the scope middleware that looks like any other auth error.